frontend/src/utilities/useAcceleratorProfileState.ts

```diff
@@ -65,9 +65,9 @@ const useAcceleratorProfileState = (
       } else {
         // check if there is accelerator usage in the container
         // this is to handle the case where the accelerator is disabled, deleted, or empty
-        const containerResourceAttributes = Object.values(ContainerResourceAttributes) as string[];
+        const containerResourceAttributes = Object.values(ContainerResourceAttributes);
         const possibleAcceleratorRequests = Object.entries(resources.requests ?? {})
-          .filter(([key]) => !containerResourceAttributes.includes(key))
+          .filter(([key]) => isEnumMember(key, containerResourceAttributes))
```

Comment:
Any idea why the original code opted to include keys that aren't in `containerResourceAttributes`? Am I reading this correctly?

Reply:
Keep the same logic as the old code. Any changes to logic should log a new issue if necessary.

Including @Gkrumbach07 since he authored this code.

Reply:
Ok, I raised this comment because I got unexpected behavior with the `isEnumMember` function. But that's because I put the array as the arg instead of the enum. Oops... Resolving.